tests/features/service-selector.feature

```diff
@@ -0,0 +1,80 @@
+
+
+Scenario: Call setItemsStream with stream of items when Selector instance is empty
+    Given Selector Service with setItemsStream method
+    When  user calls setItemsStream with a stream of items
+    Then  all items are being loaded in the Selector instance
+    And   subscribing to items method returns the stored items
+
+Scenario: Call setItemsStream with a stream of new items when Selector instance is not empty
+    Given  Selector Service with setItemsStream method
+    And    several items are stored in the Selector instance
```

Comment:
You should not assume what is stored or not and the test should not really care about the `selector instance`.
This steps can be described as 
```gherkin
Given ...
And user calls setItemsStream with stream of items A
When user calls setItemsStream with stream of items B
```